src/store/getters/state-getters.ts

```diff
@@ -96,53 +96,53 @@ export const getEntityType = (state: StateIF): CorpTypeCd => {
 }
 
 /** Whether the entity is a Benefit Company. */
-export const isTypeBEN = (state: StateIF): boolean => {
+export const isBenefitCompany = (state: StateIF): boolean => {
   return (getEntityType(state) === CorpTypeCd.BENEFIT_COMPANY)
 }
 
 /** Whether the entity is a Cooperative. */
-export const isTypeCP = (state: StateIF): boolean => {
+export const isCoop = (state: StateIF): boolean => {
   return (getEntityType(state) === CorpTypeCd.COOP)
 }
 
 /** Whether the entity is a BC Company. */
-export const isTypeBC = (state: StateIF): boolean => {
+export const isBcCompany = (state: StateIF): boolean => {
   return (getEntityType(state) === CorpTypeCd.BC_COMPANY)
 }
 
 /** Whether the entity is a Community Contribution Company. */
-export const isTypeCCC = (state: StateIF): boolean => {
+export const isBcCcc = (state: StateIF): boolean => {
   return (getEntityType(state) === CorpTypeCd.BC_CCC)
 }
 
 /** Whether the entity is an Unlimited Liability Company. */
-export const isTypeULC = (state: StateIF): boolean => {
+export const isBcUlcCompany = (state: StateIF): boolean => {
   return (getEntityType(state) === CorpTypeCd.BC_ULC_COMPANY)
 }
 
 /** Whether the entity is a Sole Proprietorship. */
-export const isTypeSP = (state: StateIF): boolean => {
+export const isSoleProp = (state: StateIF): boolean => {
   return (getEntityType(state) === CorpTypeCd.SOLE_PROP)
 }
 
 /** Whether the entity is a General Partnership. */
-export const isTypeGP = (state: StateIF): boolean => {
+export const isPartnership = (state: StateIF): boolean => {
   return (getEntityType(state) === CorpTypeCd.PARTNERSHIP)
 }
 
-/** Whether the entity is a base type (BC/BEN/CCC/ULC). */
-export const isTypeBase = (state: StateIF): boolean => {
+/** Whether the entity is of corp class BC (BC/BEN/CCC/ULC). */
+export const isCorpClassBc = (state: StateIF): boolean => {
```

Comment:
See corp class definitions in
https://github.com/bcgov/bcrs-shared-components/blob/main/src/modules/corp-type-module/corp-type-module.ts